### PR TITLE
038_ATF_N_TC_SetAudioStreamingIndicator_USER_DISALLOWED_non-media_app…

### PR DESCRIPTION
**Pull request by istoimenova**
Script is part of TS: #932

### CODE REVIEW

**Comment by istoimenova** [comment]
@OVikhrov, @dboltovskyi, please review.

**Comment by dboltovskyi** [comment]
@istoimenova , Since this script is almost the same as 037 I put review on Hold once review of 037 is finished

File: test_scripts/API/SetAudioStreamingIndicator/038_ATF_N_TC_SetAudioStreamingIndicator_USER_DISALLOWED_non-media_app_RPC_disallowed_by_user.lua [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+--
+-- Description:
+-- For Genivi applicable ONLY for 'EXTERNAL_PROPRIETARY' Polciies
+-- SDL must return 'USER_DISALLOWED, success:false' to mobile app
+-- in case the requested RPC is included to the group disallowed by the user.
+--
+-- 1. Used preconditions
+-- SDL is built with External Proprietary flag.
+-- Include in Testing_group(user_consent group) RPC SetAudioStreamingIndicator
+-- Register and activate media application
```
Comment (dboltovskyi) [inline test_scripts/API/SetAudioStreamingIndicator/038_ATF_N_TC_SetAudioStreamingIndicator_USER_DISALLOWED_non-media_app_RPC_disallowed_by_user.lua]: @istoimenova, Please correct 'media' => 'non-media'

Comment (istoimenova) [inline test_scripts/API/SetAudioStreamingIndicator/038_ATF_N_TC_SetAudioStreamingIndicator_USER_DISALLOWED_non-media_app_RPC_disallowed_by_user.lua]: @dboltovskyi, done in 3793b25691671f2a98e8d750cf6ca7c777ddf821

**Pull request merged**